gtest/spec/test_async.yaml

```diff
@@ -5,28 +5,28 @@ programs:
     path: examples/target/wasm32-unknown-unknown/release/demo_async.wasm
     init_message:
       kind: utf-8
-      value: "{2}"
+      value: "0200000000000000000000000000000000000000000000000000000000000000,0300000000000000000000000000000000000000000000000000000000000000,0400000000000000000000000000000000000000000000000000000000000000"
```

Comment:
why not {2}, {3}, {4} 🤔

Reply:
We have `<real address>,{3},{4}` after replacing in gtest. I'll fix the gtest itself.

Reply:
Fixed.